lib/blacklight/solr/search_builder_behavior.rb

```diff
@@ -282,7 +282,7 @@ def facet_value_to_fq_string(facet_field, value)
 
       case
         when (facet_config and facet_config.query)
-          facet_config.query[value][:fq]
+          facet_config.query[value].blank? ? '-*' : facet_config.query[value][:fq]
```

Comment:
Do you think this can this be simplified any?

`facet_config.query[value].blank? ? '-*' : facet_config.query[value][:fq]`